test/services/database/post_repository_test.dart

```diff
@@ -119,7 +119,7 @@ void main() {
 
     test("Post on edge (outside) is not queried", () async {
       const userPosition = userPosition1;
-      final expectedPost = FirestorePostGenerator.generatePostAt(
+      final expectedPost = FirestorePostGenerator().generatePostAt(
```

Comment:
Shouldn't we have just one instance per test ? Why recreate an instance every time we call `generatePostAt` ?

Reply:
We could maybe use `setUp` function with late assignments like we do with mocks to ensure a clean instance is used for every test ?

Reply:
I agree with this comment. @gruvw 's proposal will make this code respect our new test standards.

Reply:
My original idea was that this would be adding unnecessary boilerplate. I however understand your point of view, and apply the majority's opinion! :)
Fixed in db6722bd9a4c2218551e602b6dbc6abb6dee25bb.